src/components/common/JadeLink.tsx

```diff
@@ -0,0 +1,10 @@
+import { styled } from '@mui/system';
+import { Box } from '@mui/material';
+
+export const JadeLink = styled(Box)(({ theme }) => ({
+  ...theme.mixins.jadeLink,
+}));
```

Comment:
```suggestion
const StyledLink = styled(Box)(({ theme }: { theme: CustomTheme }) => ({
  ...theme.mixins.jadeLink,
}));
```
I think we need this odd format in order to get the right custom theme type piped through